packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1334,7 +1288,9 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
       // If we're between frames (SchedulerPhase.idle) then absorb any
       // subsequent pointers from this frame. The absorbing flag will be
       // reset in the next frame, see build().
-      final RenderAbsorbPointer absorber = _overlayKey.currentContext?.ancestorRenderObjectOfType(const TypeMatcher<RenderAbsorbPointer>());
+      final RenderAbsorbPointer absorber = _overlayKey.currentContext
+          ?.ancestorRenderObjectOfType(
```

Comment:
definitely don't line break before an elvis operator, that looks really weird :-)